app/src/main/java/org/sopt/havit/ui/share/SetNotificationFragment.kt

```diff
@@ -18,23 +17,31 @@ import java.util.*
 class SetNotificationFragment :
     BaseBindingFragment<FragmentSetNotificationBinding>(R.layout.fragment_set_notification) {
 
-    private lateinit var btnList: Array<TextView>
     private lateinit var notificationTime: String
 
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?
     ): View {
         super.onCreateView(inflater, container, savedInstanceState)
 
-        btnList = arrayOf(
-            binding.btn1h, binding.btn2h, binding.btn3h, binding.btnTomorrow, binding.btnChooseTime
-        )
-
+        initRadioGroupListener()
         initToolbarListener()
 
         return binding.root
     }
 
+    private fun initRadioGroupListener() {
+        binding.rgNotificationTime.setOnCheckedChangeListener { _, checkedId ->
+            when (checkedId) {
+                R.id.btn_1h -> getNotificationTime(ONE_HOUR)
+                R.id.btn_2h -> getNotificationTime(TWO_HOUR)
+                R.id.btn_3h -> getNotificationTime(THREE_HOUR)
+                R.id.btn_tomorrow -> getNotificationTime(TWENTY_FOUR_HOUR)
+                else -> getNotificationTime(ONE_MIN) // DatePicker Dialog 로 변경 예정
```

Comment:
```suggestion
        binding.rgNotificationTime.setOnCheckedChangeListener { _, checkedId ->
            getNotificationTime(
                when (checkedId) {
                    R.id.btn_1h -> ONE_HOUR
                    R.id.btn_2h -> TWO_HOUR
                    R.id.btn_3h -> THREE_HOUR
                    R.id.btn_tomorrow -> TWENTY_FOUR_HOUR
                    else -> ONE_MIN
```
이런식은 어떠냐고 제안하려고 했는데 DatePicker Dialog로 바꾸시는거면,, 이렇게 하고 getNotificationTime 함수에서 나눠주시거나 해야할텐데 뭔가 함수를 분리하실 것 같아서 이 제안은 별로인것같네용 ㅎㅅㅎ

Reply:
음 조금 더 설명을 덧붙이자면, 마지막 버튼의 경우에 현재는 푸시알림을 테스트  하기 위해
getNOificationTime메소드에서 1분을 더하는 로직으로 설정해뒀지만,
나중에는 커스텀 뷰를 띄울예정입니다! 그래서 suggestion 해주신 방법대로 하면,
특정 항목은 getNotificationTime을 적용하지 않도록 따로 설정을 해줘야 하는데, 그 로직이 더 복잡할듯 하여 위와 같이 짰답니다!😉